### DISCUSSION TITLE
Export all reverse-aliases created: they are our actual contact list to import in our mail client

### DISCUSSION THREAD

**Discussion by Arno0x** [simple-login/app#1819] [Feature Requests]
HI,

Could it be possible to export all the reverse-aliases created amongst all aliases. The reason being these reverse-aliases are our actual contacts list. So it would be very valuable and useful to be able to import them into every mail client (be it a web client, a heavy client on a smartphone or a computer).

I could find how to export the list of aliases, but not all the reverse-aliases created automatically or manually. To get it, I need to go on each and every alias I have, and manually copy/paste the reverse-alias to add it as a contact in my mail client. Really not convenient.

**Comment by winklerrr** [comment]
Any news on this?
My post is related: #2426
